library-extensions-expandable/src/main/java/com/mikepenz/fastadapter/expandable/ExpandableExtension.kt

```diff
@@ -22,13 +24,37 @@ fun <Item : GenericItem> FastAdapter<Item>.getExpandableExtension(): ExpandableE
  * Extension method to retrieve or create the ExpandableExtension from the current FastAdapter
  * This will return a non null variant and fail
  */
-fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
+inline fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
     getExpandableExtension().apply(block)
 }
 
+/**
+ * internal helper function to check if an item is expanded.
+ */
+internal fun IItem<out RecyclerView.ViewHolder>?.isExpanded(): Boolean {
+    return (this as? IExpandable<*>)?.isExpanded == true
+}
+
+/**
+ * internal helper function to execute the block if the item is expandable
+ */
+internal fun <R> IItem<out RecyclerView.ViewHolder>?.ifExpandable(block: (IExpandable<*>) -> R): R? {
```

Comment:
Is it possible to make this a inline function?

Reply:
I had it as an inline function. but AS will inform that the performance impact is rather low and it is suggested to keep it a normal function

Reply:
Weird. I thought all inlined higher order functions benefit from inlining (performance wise). I'm not sure if it's because of `let`, which probably inlines stuff on its own